hack/create-release.sh

```diff
@@ -31,7 +31,7 @@ REPOSITORY=NVIDIA/nvidia-container-toolkit
 echo "Creating draft release"
 ./hack/generate-changelog.sh --version ${VERSION} | \
     grep -v "### Version v" | \
```

Comment:
generate-changelog doesn't have a string `### Version v`,  this match won't happen

Reply:
You're right. It's an inverted match anyway, so we'll skip something that's not there.